aws/sdk/integration-tests/kms/tests/sensitive-it.rs

```diff
@@ -55,7 +55,12 @@ async fn client_is_debug() {
 async fn client_is_clone() {
     let conf = kms::Config::builder().build();
     let client = kms::Client::from_conf(conf);
-    let _ = client.clone();
+
+    fn is_clone(it: impl Clone) {
+        drop(it)
+    }
+
+    is_clone(client);
```

Comment:
Clippy was mad that we cloned and didn't use the client it but it's fine if we use a function and drop the client.